frontend/src/functions/services/api/hours_request/stats.js

```diff
@@ -0,0 +1,8 @@
+export const getHoursStats = (obj) => {
+    let url = `/api/hours/stats/`;
+    fetch(url)
+        .then(response => response.json())
+        .then(data => {
+            obj.setState({chartData:data});
+        }).catch(err => console.log(err))
```

Comment:
this is the kind of console log you should leave -> it only happens when an error occurred